src/Runner.Sdk/BuildConstants.cs

```diff
@@ -1,19 +0,0 @@
-namespace GitHub.Runner.Sdk
```

Comment:
we don't want to delete the dummy file from the source code, I think.

Reply:
Since it's generated on build, I thought it fine to remove and ignore it. 
If I don't delete it, ignoring it does nothing and we're back to having it in the diff every time we build.